flexmeasures/data/scripts/cli_tasks/data_add.py

```diff
@@ -305,16 +312,21 @@ def add_beliefs(
     if sensor is None:
         print(f"Failed to create beliefs: no sensor found with id {sensor_id}.")
         return
-    source = (
-        DataSource.query.filter(DataSource.name == "Seita")
-        .filter(DataSource.type == "CLI script")
-        .one_or_none()
-    )
-    if not source:
-        print("SETTING UP CLI SCRIPT AS NEW DATA SOURCE...")
-        source = DataSource(name="Seita", type="CLI script")
-        db.session.add(source)
-        db.session.flush()  # assigns id
+    query = DataSource.query.filter(DataSource.type == "CLI script")
```

Comment:
I'd like this as its own utility function (`get_or_create_source(source: Union[int, str])`), maybe in `data.models.data_sources`.

I want to use it from the customer script I am working on right now, which reads in measurements per third-party API.

Reply:
Maybe the `flush` could be optional (defaults to True)

Reply:
I just found out we have `api_utils.get_or_create_user_data_source(user: User)` which already looks quite similar.
Maybe move this out of the `api` module (to `data.models.data_sources`) and make it more general?